deploy/chart/templates/0000_50_olm_15-packageserver.clusterserviceversion.yaml

```diff
@@ -0,0 +1,74 @@
+{{ if (.Values.installType) and eq .Values.installType "ocp" }}
+# keep in sync with _packageserver.clusterserviceversion.yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: packageserver.v{{ .Chart.Version }}
```

Comment:
Nice!

Reply:
Is this changed in the `olm-operators` `ConfigMap` too?

Reply:
Should be. I didn't actually change _packageserver.clusterserviceversion.yaml, which is included in the configmap.